docs/designers-developers/developers/data/data-core.md

```diff
@@ -148,6 +148,28 @@ Return Upload Permissions.
 
 Upload Permissions.
 
+### canUser
+
+Returns whether the current user can perform the given action on the given
+REST resource.
+
+Calling this may trigger an OPTIONS request to the REST API via the
+`canUser()` resolver.
+
+https://developer.wordpress.org/rest-api/reference/
+
+*Parameters*
+
+ * state: Data state.
+ * action: Action to check. One of: 'create', 'read', 'update',
+                          'delete'.
```

Comment:
> This indentation is kinda odd; I'm not sure why this is indented where it is...

Likely because it's unprocessed from the space-aligned JSDoc. Could probably be improved with some additional processing (`lines.map( trim ).join( ' ' )`?). It has no difference in the Markdown preview though, since the whitespace is collapsed.